packages/rancher-sachet/package.yaml

```diff
@@ -1,2 +1 @@
 url: local
-version: 100.0.0
```

Comment:
Since it is a local chart, so we can change its version in the `Chart.yaml` directly. Otherwise, the chart version will be wrong inside the `rancher-alerting-drivers/charts/sachet` sub-chart

Reply:
https://github.com/rancher/charts/blob/33db4185684ece7b2ca1575cdc66af4841967764/packages/rancher-istio/package.yaml#L2 so this is how we handled istio since it is a local chart - is there a consistent way to handle this that we should document for local charts?

Reply:
@jiaqiluo I don't think it makes a difference if the version is incorrect in the subchart since no user will ever see it and we refer to sachet as a local chart. Nonetheless, I think we should just keep the version in the package.yaml since that's what we shipped in 2.6.0

Reply:
@brendarearden for most local charts, +up is not required; Istio is an edge case in that you have an upstream set of manifests, so you are manually patching the version.

I don't think there is any general, consistent guidance we would need but what would you propose?

Reply:
@aiyengar2 my example was just meant to show that the version was still in package.yaml for istio (didn't mean for the +up to throw the example off) and not removed completely, so it sounds like that is consistently what we want to do . 

Mainly I was just concerned that if we were going to have charts that didn't have versions in the package.yaml then we would want to document in the readme why that is. Sounds like we will not be doing that so I think the current documentation is pretty good as is.

Reply:
Thank you @brendarearden and @aiyengar2 for bringing this question up! 
I agree with you two that we should keep the version in the `package.yaml` as the source of truth for the chart version.